src/angular/header-lean/app-chooser-section.scss

```
@@ -47,4 +47,25 @@
   &:last-child {
     border-bottom: var(--sbb-border-width-thin) solid var(--sbb-color-cloud);
   }
+
+  .sbb-icon,
+  img {
+    vertical-align: middle;
+    width: var(--sbb-icon-size-default);
+    height: var(--sbb-icon-size-default);
+
+    // adjust margin to keep line height
+    margin-top: calc((var(--sbb-line-height-medium) - var(--sbb-icon-size-default)) / 2);
+    margin-bottom: calc((var(--sbb-line-height-medium) - var(--sbb-icon-size-default)) / 2);
+
+    svg {
+      width: 100%;
+      height: 100%;
+    }
+
+    &:first-child {
+      float: left; // used to ignore users whitespace and to ensure margin is correct
```

```suggestion
      float: left; // Used to ignore users whitespace and to ensure margin is correct
```
We once started to write comment starting letters upper case

---

Good to know. However, I've removed the float after our current discussion.